deb/brave-keyring/DEBIAN/postrm

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+
+# If the old keyring file is a symlink created in postinst, delete it.
+if [ -L /etc/apt/trusted.gpg.d/brave-browser-release.gpg ]; then
```

Comment:
Nit: the `if` is likely unnecessary. Either the file is from an old package (in which case it should be deleted) or it's the symlink we added (it should also be deleted). I can't think of a case where we'd want an old keyring to stick around when the `brave-keyring` package is removed.

Reply:
Fixed.